sdk/avalon_sdk/work_order/work_order_params.py

```diff
@@ -42,16 +44,19 @@ def __init__(
         self.set_workload_id(workload_id)
         self.set_requester_id(requester_id)
         if worker_encryption_key:
-            self.set_worker_encryption_key(
-                worker_encryption_key.encode("UTF-8").hex())
+                self.set_worker_encryption_key(
+                    worker_encryption_key)
         if data_encryption_algorithm:
```

Comment:
Can we add validation for data_encryption_algorithm? It should be a string value. 
QA team raised this concern, if they pass list of algorithms from generic client, it was still going fine.

Reply:
done